### PR TITLE
Added Polish locale

### PR DESCRIPTION
**Pull request by skill3472**
Added Polish translation. The other PR has a typo, so please don't accept #1

### CODE REVIEW

File: locale/en/pl.cfg [hunk 1]
```diff
@@ -0,0 +1,26 @@
+[entity-name]
+classic-nixie-tube=Lampa Jarzeniowa
```
Comment (pindab0ter) [inline locale/en/pl.cfg]: Do you think "[Lampa cyfrowa](https://pl.wikipedia.org/wiki/Lampa_cyfrowa)" is a better translation? Apparently "Jarzeniowa" is "fluorescent lamp", which a Nixie tube is not.

**Review by pindab0ter — CHANGES_REQUESTED** [review]
FYI, you can fix things with a new commit and push it to he same branch, this will update the PR so you don't have to make a new one for every change.

**Comment by pindab0ter** [comment]
Your PR inspired me to improve the localisation and add Dutch as well. Please review the changes to the [`en.cfg`](https://github.com/pindab0ter/UPSFriendlyNixieTubeDisplay/commit/8c182590d0432ab9fd9eb71b81c33138b42601b3) file.

**Comment by skill3472** [comment]
Hey, the full name for a nixie tube is Jarzeniowa Lampa Cyfrowa, so that's
just a shortening we use in common language - I think it's more descriptive
here, than the other options for a nixie tube that I checked. I also
verified this with 2 of my friends, we think it's the best translation :)
But it's your fork, if you wanna change it, you can!
PS. You can check out the valid translations here, but the NIXIE one doesn't work too well - 95% of even the more technical people don't know this term here.
https://pl.m.wikipedia.org/wiki/Lampa_cyfrowa

**Comment by pindab0ter** [comment]
Hey, thank you for your explanation! I was just doing a sanity check. I don't speak Polish, so if you say you and your friends think this is the right translation, then that's good enough for me!

File: locale/en/pl.cfg
Comment (pindab0ter) [inline locale/en/pl.cfg:1]: The file should be in `locale/pl/pl.cfg`, not in `locale/en/pl.cfg`.

**Review by pindab0ter — CHANGES_REQUESTED** [review]

File: locale/en/pl.cfg
Comment (pindab0ter) [inline locale/en/pl.cfg:1]: Please update the translations to match the changes I've made to [`locale/en/en.cfg`](https://github.com/pindab0ter/UPSFriendlyNixieTubeDisplay/blob/34896217418c5f13c1c2bc6495fe1aad0e614233/locale/en/en.cfg)

**Review by pindab0ter — CHANGES_REQUESTED** [review]

File: locale/en/pl.cfg
Comment (skill3472) [reply locale/en/pl.cfg:1]: Yessir! I'll do all that after work today, thanks for the tip.

**Comment by skill3472** [comment]
Hey there! Back from work, I think everything is up to the new specifications. Please review :)
Cheers!

**Review by pindab0ter — APPROVED** [review]

**Review by pindab0ter — APPROVED** [review]

**Comment by pindab0ter** [comment]
Thank you for your contribution!

I updated the mod, so you should be able to enjoy your Polish translation now.

**Pull request merged**